pointcloud_topic:
    camera_topic: /velodyne_points

reference_frame:
    frame_id: base_footprint

fixed_frame:
    frame_id: map
    
transform:
    position:
        x: -0.3
        y: 0.0
        z: 1.45
    orientation:
        roll: 0.0
        pitch: 0.279253
        yaw: 0.0

voxel_grid:
    x: 0.01
    y: 0.01
    z: 0.01
    enable: false
    
pass_through:
    x_min: -2.0
    x_max: 3.0
    x_enable: true
    y_min: -2.0
    y_max: 2.0
    y_enable: true
    z_min: 0.0
    z_max: 3.0
    z_enable: true

segmentation:
    distance_threshold: 0.01
    enable: false
    
outlier_removal:
    meanK: 50
    standard_dev_mult: 1.0
    enable: false
    
cluster_extraction:
    cluster_tolerance: 0.2
    min_cluster_size: 20
    max_cluster_size: 500
    enable: false
    
boundingbox:
    offset: 0.2
    oriented: false